internal/service/organization/resource_organization.go

```diff
@@ -88,7 +89,7 @@ func resourceCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	conn := meta.(*config.MongoDBClient).AtlasV2
 	organization, resp, err := conn.OrganizationsApi.CreateOrganization(ctx, newCreateOrganizationRequest(d)).Execute()
 	if err != nil {
-		if resp != nil && resp.StatusCode == http.StatusNotFound {
+		if resp.StatusCode == http.StatusNotFound && !strings.Contains(err.Error(), "USER_NOT_FOUND") {
 			d.SetId("")
 			return nil
 		}
```

Comment:
with the existing implementation, would the creation still fail in the line below?
`return diag.FromErr(fmt.Errorf("error creating Organization: %s", err))`

just curious why the customer in the HELP ticket did not see this error message.

Reply:
Since the status code =404, the function did `d.SetId("")` and returned nil, producing the Inconsistent Provider Result